index.html

```diff
@@ -27,21 +27,163 @@ <h4 class="dev">I’m a Software Developer</h4>
 
       <ul class="social-media">
         <li>
-          <a href="#"><img src="./Vector.png " alt=" " /></a>
+          <a href="#"><img src="./img/Vector.png " alt=" " /></a>
         </li>
         <li>
-          <a href="#"><img src="./Group.svg" alt="" /></a>
+          <a href="#"><img src="./img/Group.svg" alt="" /></a>
         </li>
         <li>
-          <a href="#"><img src="./Vector1.svg" alt="" /></a>
+          <a href="#"><img src="./img/Vector1.svg" alt="" /></a>
         </li>
         <li>
-          <a href="#"><img src="./Vector2.svg " alt="" /></a>
+          <a href="#"><img src="./img/Vector2.svg " alt="" /></a>
         </li>
         <li>
-          <a href="#"><img src="./Vector.svg" alt="" /></a>
+          <a href="#"><img src="./img/Vector.svg" alt="" /></a>
         </li>
       </ul>
     </main>
+    <h2 class="titel">My Recent Works</h2>
+    <hr />
+    <section class="works">
+      <a class="placeholder">
+        <img src="./img/ImgPlaceholder.png" alt="" />
```

Comment:
[OPTIONAL]

- [ ] Please know that the link here for the image is wrong it should be `./img/Img Placeholder.png`, to avoid issues like this I suggest that you name your files with `snake_case` in this case it will be like this `img_placeholder.png`. 😁